hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py

```diff
@@ -22,8 +28,20 @@ def perform_enque_for_review(match_message: MatchMessage):
 
 
 def perform_action(match_message: MatchMessage, action_label: ActionLabel):
-    # TODO implement
-    logger.debug("perform action")
+    action_performer = get_action_perfromers_config()[action_label]
+    action_performer.perform_action(match_message)
+
+
+def get_action_perfromers_config() -> t.Dict[ActionLabel, ActionPerformer]:
```

Comment:
I like this approach, it's a great way to define the interfaces and unblock work, while someone else can iterate on the storage and config separately.